js/src/edit-block/components/settings/textarea-default.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Converts the default setting to a string.
+ *
+ * @param {Array|*} initial The default setting as an array, or another type that will be returned.
+ * @return {string|*} The setting converted to a string, or if passed a non-array, the same type it was passed.
+ */
+const convertDefaultArrayToString = ( initial ) => {
+	return Array.isArray( initial )
+		? initial.join( '\n' )
+		: initial;
+};
+
+/**
+ * Converts the default setting to an array.
+ *
+ * @param {string|*} initial The settings default as a string, or another type that will be returned.
+ * @return {Array|*} The setting converted to an array, or if passed a non-string, the same type it was passed.
+ */
+const convertDefaultStringToArray = ( initial ) => {
+	return 'string' === typeof initial
+		? initial.split( /\r\n|[\r\n]/ )
+		: initial;
+};
+
+/**
+ * @typedef {Object} TextareaDefaultProps The component props.
+ * @property {Function} handleOnChange Handles a change in this setting.
+ * @property {import('../editor').Setting} setting This setting.
+ * @property {Array|undefined} value The setting value.
+ */
+
+/**
+ * The textarea default component.
+ *
+ * @param {TextareaDefaultProps} props The component props.
+ * @return {React.ReactElement} The textarea default component.
+ */
+const TextareaDefault = ( { handleOnChange, setting, value } ) => {
```

Comment:
This component should fix the problem with the Multiselect error. It saves the default value as an array of strings, instead of the array of objects it did before.